doc/RELEASE_PROCESS.rst

```diff
@@ -78,69 +74,47 @@ This is an outline of what needs to be done in order to release rst2pdf.
     ::
 
        $ echo -e "[egg_info]\ntag_build=rc1\n" > /tmp/build_opts.cfg
-       $ DIST_EXTRA_CONFIG=/tmp/build_opts.cfg python -m build
+       $ DIST_EXTRA_CONFIG=/tmp/build_opts.cfg uv build
        $ rm /tmp/build_opts.cfg
 
     If you're doing an alphaX, betaX or postX, then change ``tag_build=rc1`` appropriately
 
-#. Set up PyPI if you haven't already
-
-    Create a ``~/.pypirc`` file with sections for rst2pdf and testrstpdf which are then used with ``twine`` via the
-    ``--repository`` switch.
-
-    It should contain the following:
-
-    ::
-
-        [rst2pdf]
-          username = __token__
-          password = {your token here}
-
-        [testrst2pdf]
-          repository = https://test.pypi.org/legacy/
-          username = __token__
-          password = {your token here}
-
-
-    You can get your token from the account settings section of https://pypi.org/ & https://test.pypi.org/
```

Comment:
Do we need to keep the instructions for getting the token as I think this is the one used in the new command as well?

Reply:
Yes we do. I've re-added.